tap_dynamics_bc/streams.py

```diff
@@ -473,43 +473,8 @@ class VendorPaymentJournalsStream(dynamicsBcStream):
         th.Property("company_name", th.StringType),
     ).to_dict()
 
-    def get_child_context(self, record, context):
-        return {
-            "company_id": context["company_id"], 
```

Comment:
Doesn't removing this from the child context cause issues for other streams where we reference the `company_id`?